en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-traffic.md

```diff
@@ -1,2 +1,57 @@
+# Traffic Page
```

Comment:
@tishan89 - I applied a commit directly to this page.
Please check before merging.

Reply:
ack